.github/workflows/build.yml

```diff
@@ -54,6 +54,10 @@ jobs:
         run: |
           git clone -b dev --depth 1 https://github.com/open-mmlab/mmyolo.git /home/runner/work/mmyolo
           python -m pip install -v -e /home/runner/work/mmyolo
+      - name: Install mmpose
+        run: |
```

Comment:
may update the en/zh-cn docs as well
https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmpose.html#supported-models